### ISSUE TITLE
Installed successfull, but can not be used.

### ISSUE THREAD

**Issue by amormiosl** [cDima/Aerial#1]
I right clicked Aerial.scr and choose Install, there was no error occured.

But under the Screen Saver Settings, I didn't found any screensaver named AerialScreenSaver.

What's the problem?

BTW, my os is Win7 64-bit.

**Comment by cDima** [comment]
When you click install, the screen saver window should popup with the screensaver selected. This doesn't happen on your machine?

**Comment by cDima** [comment]
@amormiosl also please check that you have installed the .net framework 4.6 https://support.microsoft.com/en-us/kb/2715633 and have the windows media player installed (default) on win7.

**Comment by edweerdt** [comment]
I might have the same problem. Choosing install opens the screensaver menu with Aerial showing. I can change the options and successfully preview the screensaver. However,if I close the screensaver settings and open it again, Aerial is not listed anymore as a screensaver.

**Comment by cDima** [comment]
When installing are you unzipping it in a secure location or somewhere temporary? Perhaps it gets deleted after it is setup.

**Comment by edweerdt** [comment]
It is in a seperate folder with read and write access. The file is still there.

**Comment by danteali** [comment]
When I tried to run it 2 days ago (Friday 6th Nov) I got the error: "ScreenSaver has stopped working" and the details show problem event name CLR20r3. 
[I do have .NET framework 4.6 and media player installed]

Tried downloading again today (Sun 8th Nov) in case and further updates had been made to the program. Now, once the zip is extracted, the file gets detected by Windows Security Essentials as a trojan (Win32/Spallowz.A!plock) and deleted. I took the chance of restoring it from the Security Essentials vault and running it and I still get the error above.
